testsuite/features/build_validation/migration/salt_migration_minion_migration.feature

```diff
@@ -1,29 +1,29 @@
-# Copyright (c) 2023 SUSE LLC
+# Copyright (c) 2024 SUSE LLC
```

Comment:
```suggestion
# Copyright (c) 2023-2024 SUSE LLC
```
You need to include the year when this was created as well so I'd put it as a range

Reply:
Right, thanks